src/main/java/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCM.java

```diff
@@ -543,63 +526,66 @@ public boolean getShowGitToolOptions() {
          * @throws FormException if any data is missing
          */
         @Override
-        public SCM newInstance(@Nullable StaplerRequest req, JSONObject formData)
-                throws FormException {
-            try {
-                BitbucketSCM scm = (BitbucketSCM) super.newInstance(req, formData);
-                scm.setBitbucketClientFactoryProvider(bitbucketClientFactoryProvider);
-                scm.setBitbucketPluginConfiguration(bitbucketPluginConfiguration);
-
-                String serverId = formData.getString("serverId");
-                BitbucketServerConfiguration serverConf = bitbucketPluginConfiguration.getServerById(serverId)
-                        .orElseThrow(() -> new FormException("No server with ID " + serverId, "serverId"));
-
-                String credentialsId = formData.getString("credentialsId");
-                BitbucketCredentials credentials = createWithFallback(CredentialUtils.getCredentials(credentialsId), serverConf);
-                BitbucketClientFactory clientFactory = bitbucketClientFactoryProvider.getClient(serverConf.getBaseUrl(), credentials);
-
-                String projectName = formData.getString("projectName");
-                if (isBlank(projectName)) {
-                    throw new BitbucketSCMException("A project is required", "projectName");
-                }
+        public SCM newInstance(@Nullable StaplerRequest req, JSONObject formData) throws FormException {
+            BitbucketSCM scm = (BitbucketSCM) super.newInstance(req, formData);
+            scm.setBitbucketClientFactoryProvider(bitbucketClientFactoryProvider);
+            scm.setBitbucketPluginConfiguration(bitbucketPluginConfiguration);
+
+            String serverId = formData.getString("serverId");
+            if (isBlank(serverId)) {
+                LOGGER.severe("Error creating the Bitbucket SCM: The serverId cannot be blank");
```

Comment:
I think this is too high level, it is the result of user input.
The mapping between slf4j that we're used to and the java.util logging is as follows:
```
FINEST  -> TRACE
FINER   -> DEBUG
FINE    -> DEBUG
CONFIG  -> INFO
INFO    -> INFO
WARNING -> WARN
SEVERE  -> ERROR
```
(from https://www.slf4j.org/apidocs/org/slf4j/bridge/SLF4JBridgeHandler.html) based on that I think most of these should be under fine, or info. Only a few rare ones are higher than that. What do you think?

Reply:
I was downgrading it from throwing an exception, which is why I felt that severe was reasonable. Its not just a user error - the entire configuration is going to be broken going forwards. If my config is broken and I can't figure out why, I would consider looking in the logs